src/Security.php

```diff
@@ -24,296 +26,50 @@
  * @author Derek Jones <derek.jones@ellislab.com>
  * @author Graham Campbell <graham@alt-three.com>
  */
-class Security
+final class Security
 {
     /**
-     * The evil attributes.
-     *
-     * @var string[]
+     * @var AntiXSS
      */
-    protected $evil;
+    private $antiXss;
 
     /**
      * Create a new security instance.
      *
      * @param string[]|null $evil
+     * @param string        $replacement
```

Comment:
Please have defaults to `null` (and have the constructor convert null to the actual default).

Reply:
done